website/src/components/Changelog/Entries.tsx

```diff
@@ -102,6 +102,12 @@ function Previous({
   );
 }
 
+type EntryProps = {
+  version: string;
+  date: Date;
+  children: React.ReactNode;
```

Comment:
There is a composite type that adds children for you. `ReactNodeWithChildren` or something like that.

Reply:
`PropsWithChildren` 👍